examples/material_gallery/test/example_code_parser_test.dart

```diff
@@ -0,0 +1,77 @@
+// Copyright 2016 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+
+import 'package:flutter_test/flutter_test.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+
+import 'package:mojo/core.dart' as core;
+
+import 'package:test/test.dart';
+
+import '../lib/gallery/example_code_parser.dart';
+
+void main() {
+  test('Material Gallery example code parser test', () {
+    testWidgets((WidgetTester tester) {
+      TestParsingWidget testWidget = new TestParsingWidget();
```

Comment:
you can simplify this test by inlining the test part (the two calls to getExampleCode) into the main(). Actually you don't even need to use testWidgets() since this doesn't actually test any widgets.

Also you should use the async features of the 'test' package. Right now if getExampleCode throws, or if it never completes, you'll never find out if the test fails. (In fact I'm not convinced that the way the test is written now, it'll actually ever fail. Are you sure the expect() calls are being executed?)